src/main/groovy/org/exoplatform/platform/am/settings/AddonsManagerSettings.groovy

```diff
@@ -44,7 +47,12 @@ class AddonsManagerSettings extends Properties {
    * Automatically load properties for {@link AddonsManagerSettings#ADDONS_MANAGER_PROPERTIES}
    */
   protected void init() {
-    InputStream inputStream = getClass().getClassLoader().getResourceAsStream(ADDONS_MANAGER_PROPERTIES)
+    InputStream inputStream = null;
+    if (System.getProperty("am.properties.path") != null) {
```

Comment:
Shoudn't we call the property "meeds.am.properties.path" to make it clear it is related to meeds ?

Reply:
In fact, it's not for meeds only, but even for eXo, so I prefer to not add product prefix here.

Reply:
eXo will use Meeds addon manager actually, so it seems logical to add meeds prefix to me. My concern is that the name "am.properties.path" sounds quite "generic", it would help ops to have a more meaningful name IMO. Or at least it should be addons-manager.properties.path.

Reply:
ok, for `addons-manager.properties.path`, done.
In fact, it will not be used by ops. Just in distribution and it will not be changed (at all I think).